docs/vgclog_balancedexamples.md

```diff
@@ -0,0 +1,643 @@
+
+```xml
+<gc-op id="189" type="copy forward" timems="400.637" contextid="186" timestamp="2021-02-26T11:11:42.713">
+  <memory-copied type="eden" objects="4434622" bytes="119281928" bytesdiscarded="1382272" />
+  <memory-copied type="other" objects="8847813" bytes="244414264" bytesdiscarded="6243176" />
+  <memory-cardclean objects="1446970" bytes="64143048" />
+  <regions eden="512" other="80" />
+  <remembered-set-cleared processed="2435794" cleared="887129" durationms="8.667" />
+  <finalization candidates="66" enqueued="56" />
+  <ownableSynchronizers candidates="256500" cleared="78012" />
+  <references type="soft" candidates="153648" cleared="0" enqueued="0" dynamicThreshold="22" maxThreshold="32" />
+  <references type="weak" candidates="1266" cleared="610" enqueued="430" />
+  <stringconstants candidates="9479" cleared="0"  />
+  <object-monitors candidates="13576" cleared="13505"  />
+</gc-op>
+<gc-op id="190" type="classunload" timems="0.010" contextid="186" timestamp="2021-02-26T11:11:42.713">
+  <classunload-info classloadercandidates="179" classloadersunloaded="0" classesunloaded="0" anonymousclassesunloaded="0" quiescems="0.000" setupms="0.010" scanms="0.000" postms="0.000" />
+</gc-op>
+```
+
+The logs show that the copy forward operation acts on the the entire *eden* space (512 regions), recorded as `type=eden`, and 80 older regions, which are recorded as `type=other`. 113.76 MB (119281928 B) have been copied from the *eden* space to 1st generation regions and 233.10 MB (244414264 B) of memory in non-*eden* regions have been copied to the next generation of regions. The copy forward operation is followed by a class unload operation.
+
+The element `<gc-end>` records the end of the increment and provides another snapshot of memory allocation on the heap, similar to `<gc-start>`.
+
+```xml
+<gc-end id="191" type="partial gc" contextid="186" durationms="402.645" usertimems="3157.520" systemtimems="4.000" stalltimems="47.689" timestamp="2021-02-26T11:11:42.714" activeThreads="8">
+  <mem-info id="192" free="3003121664" total="4294967296" percent="69">
+    <mem type="eden" free="2147483648" total="2147483648" percent="100" />
+    <arraylet-primitive objects="1" leaves="4" largest="4" />
+    <pending-finalizers system="56" default="0" reference="430" classloader="0" />
+    <remembered-set count="2922048" freebytes="160016128" totalbytes="171704320" percent="93" regionsoverflowed="1" regionsstable="12" regionsrebuilding="0"/>
+  </mem-info>
+</gc-end>
+
+```
+
+The heap memory allocation at the end of the increment is as follows:
+
+- The heap now has 2864 MB(3003121664 B) of memory available compared to the 856 MB available at the start of the increment. The increment reclaimed 2008 MB of memory in the heap. 
+
+- 100% of the *eden* space is available as free memory. All the used memory in the *eden* space is reclaimed.
```

Comment:
well, the last stentence is not fully correct.
we reclaimed slightly less then eden, but we used some other free regions to recreate eden.